debezium-embedded/src/test/java/io/debezium/embedded/AbstractConnectorTest.java

```diff
@@ -279,6 +293,16 @@ protected void start(Class<? extends SourceConnector> connectorClass, Configurat
             LoggingContext.forConnector(getClass().getSimpleName(), "", "engine");
             engine.run();
         });
+        try {
+            if (!latch.await(10, TimeUnit.SECONDS)) {
+                // maybe it takes more time to start up, so just log a warning and continue
+                logger.warn("The engine startup has not completed in the expected amount of time;");
```

Comment:
May want to say "The connector did not finish starting its task(s) or complete in the expected amount of time."